src/scenes/ProgressReports/Detail/ProgressReportDrawer/RoleIcon.tsx

```diff
@@ -0,0 +1,56 @@
+import { Grading, Translate } from '@mui/icons-material';
+import { Box } from '@mui/material';
+import { Role } from '~/api/schema.graphql';
+import { PeopleJoinedIcon } from '~/components/Icons';
+import { ProjectManagerIcon } from '~/components/Icons/ProjectManagerIcon';
+import { colorPalette } from './colorPalette';
+
+interface StepIconProps {
+  roleStep?: Role | null;
+}
+
+const variantToIconMapper: {
+  [key in Role]: React.FC | null;
+} = {
+  FieldPartner: PeopleJoinedIcon,
+  Translator: Translate,
+  ProjectManager: ProjectManagerIcon,
+  Marketing: Grading,
+  Administrator: null,
+  BetaTester: null,
+  BibleTranslationLiaison: null,
+  Consultant: null,
+  ConsultantManager: null,
+  Controller: null,
+  ExperienceOperations: null,
+  FieldOperationsDirector: null,
+  FinancialAnalyst: null,
+  Fundraising: null,
+  Intern: null,
+  LeadFinancialAnalyst: null,
+  Leadership: null,
+  Liaison: null,
+  Mentor: null,
+  RegionalCommunicationsCoordinator: null,
+  RegionalDirector: null,
+  StaffMember: null,
+};
+
+export const RoleIcon = ({ roleStep = 'FieldPartner' }: StepIconProps) => {
```

Comment:
I don't think this component should assume any default. It's a business-like decision that should be made in a smarter component.